price-feeder/oracle/client/client.go

```diff
@@ -161,16 +143,15 @@ func (oc OracleClient) BroadcastVote(nextBlockHeight int64, timeoutHeight int64,
 
 		err = tx.BroadcastTx(ctx, factory, msgs...)
```

Comment:
So this isn't related to the PR, but I looked at this function and it's 99% of what we want. That method is meant to be used by the CLI.

The reason I bring this up is because we want the response object of the tx, so we can for example log the tx hash and block it was committed in.

Let's address this in a separate PR though, i.e. copy the entire contents of this method except the last line where we print it.